index.html

```diff
@@ -1,75 +1,287 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Proiect HTML+CSS</title>
+<html lang="ro">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MM Atelier</title>
     <style>
-        body {
-            margin: 0;
-            min-height: 100vh;
-        }
-    </style>
-</head>
-<body>
-    <h1>Web Heroines</h1>
+      body {
+        margin: 0;
+        padding: 0;
+        height: 100rem;
+      }
+      @font-face {
+        font-family: lemonmilk;
+        src: url("font/LemonMilk.woff") format("webopenfontformat"),
+          url("font/LemonMilk.ttf") format("truetype");
+        font-variant-ligatures: none;
+      }
+      @font-face {
+        font-family: modern;
+        src: url("font/Modern.woff") format("webopenfontformat"),
+          url("font/Modern.ttf") format("truetype");
+      }
+      .banner {
+        width: 100%;
+        height: 100vh;
+        overflow: visible;
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        align-items: center;
+      }
+      .banner video {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100vh;
+        object-fit: cover;
+        pointer-events: none;
+      }
+      .banner .title {
+        z-index: 1;
+        margin: 0 auto;
+        text-align: center;
+      }
+      .banner .title h1 {
+        font-family: "modern";
```

Comment:
Când folosești un webfont, e bine să îi dai și fallback.

```suggestion
        font-family: "modern", sans-serif;
```